gordo_components/model/factories/feedforward_autoencoder.py

```diff
@@ -1,24 +1,27 @@
 # -*- coding: utf-8 -*-
 
-from typing import List, Tuple, Dict, Any
+from typing import Tuple, Dict, Any
 
 from keras import regularizers
 from keras.layers import Dense
 import keras
 
 from keras.models import Sequential as KerasSequential
 from gordo_components.model.register import register_model_builder
-from gordo_components.model.factories.model_factories_utils import hourglass_calc_dims
+from gordo_components.model.factories.model_factories_utils import (
```

Comment:
To follow the same pattern, you can just name this `utils`; as that's what we have in most of our component modules.

Reply:
This goes along with the last comment, but this was already there. Is it best for me to rename it as I come across it and include it in my PR, create a new issue/PR, etc?